api4/system_test.go

```diff
@@ -570,7 +570,7 @@ func TestGetAnalyticsOld(t *testing.T) {
 	CheckUnauthorizedStatus(t, resp)
 }
 
-func TestS3TestConnection(t *testing.T) {
+/*func TestS3TestConnection(t *testing.T) {
```

Comment:
My concern here is, was it a temporary fix that mustn't be merged into master or not?

Reply:
It was a temporary fix to get the RC cut, so I think you may be right. I will put this test back in for master.